### DISCUSSION TITLE
Styling of the video component and container

### DISCUSSION THREAD

**Discussion by yuchuga** [JodusNodus/react-qr-reader#275] [Q&A]
Hi [JodusNodus](https://github.com/react-qr-reader/react-qr-reader/commits?author=JodusNodus) how do I fill the video component to the entire space of the container since we are using the props videoStyle and videoContainerStyle? Also, how do we adjust the size of the videoContainer to fill up a certain portion on the browser. Are we also able to use className prop in <QrReader> component?

        <QrReader
          onResult={handleScan}
          delay={100}
          videoStyle={previewStyle}
          videoContainerStyle={camStyle}
        />

const previewStyle = {
  display: 'flex',
  justifyContent: 'center',
  width: '100%',
  height: '100%',
}

const camStyle = {
  display: 'flex',
  justifyContent: 'center',
  marginTop: '-100px',
}

**Comment by Zach-EE** [comment]
did you ever figure out a solution to this?
